blueprints/ember-cli-blanket/files/tests/blanket-options.js

```diff
@@ -8,7 +8,8 @@ var options = {
   enableCoverage: true,
   cliOptions: {
     reporters: ['json']
-  }
+  },
+  autostart: true
```

Comment:
Good point, adjusted the location - also added a guard in case the property isn't defined.